ledger-service/http-json/src/main/scala/com/digitalasset/http/domain.scala

```diff
@@ -53,15 +53,26 @@ object domain {
   private def oneAndSet[A](p: A, sp: Set[A]) =
     OneAnd(p, sp - p)
 
+  trait JwtPayloadTag
+
+  trait JwtPayloadG[T[_]] {
+    val ledgerId: LedgerId
+    val applicationId: ApplicationId
+    val readAs: List[Party]
+    val actAs: T[Party]
+    val parties: OneAnd[Set, Party]
+  }
```

Comment:
```suggestion
  trait JwtPayloadTag

  trait JwtPayloadG {
    val ledgerId: LedgerId
    val applicationId: ApplicationId
    val readAs: List[Party]
    val actAs: List[Party]
    val parties: OneAnd[Set, Party]
  }
```
and then rename the field in `JwtWritePayload` to `submitter` or something and define `actAs` to return the list. Seems like a much simpler and also more useful abstraction since you can actually use it generically with something more htan calling `toSrting` on it.

Reply:
True, that is more useful :thinking: Will do

Reply:
Adressed with 8ec7126